### ISSUE TITLE
Support inline images?

### ISSUE THREAD

**Issue by defunkt** [bdewey/org-ruby#2]
Not sure if I'm just missing it, but does org-ruby support inline images?

The [org manual](http://orgmode.org/manual/Images-and-tables.html#Images-and-tables) says:

```
Some backends (HTML, LaTeX, and DocBook) allow you to directly include images into the exported document. Org does this, if a link to an image files does not have a description part, for example [[./img/a.jpg]].
```

I can't find any specs referencing images, and can't get the `[[./img/a.jpg]]` syntax to spit out anything but hyperlinks.

An example document trying to embed images is (http://github.com/dharmatech/agave/blob/master/agave.org)

Thanks!

**Comment by bdewey** [comment]
No, it doesn't support inline images (right now). I'll see if I can spin a version with support soon. FYI -- image support should be easy, caption/label support for images will be harder.

**Comment by bdewey** [comment]
OK, resolved in Version 0.5.3 of the gem.

**Comment by defunkt** [comment]
Sweet!
